test/ffmpeg-qsv/transcode/avc.py

```diff
@@ -0,0 +1,182 @@
+###
+### Copyright (C) 2018-2019 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+from ....lib import *
+from ..util import *
+from .transcoder import TranscoderTest
+
+spec = load_test_spec("avc", "transcode")
+class to_avc(TranscoderTest):
+  @slash.requires(have_ffmpeg_h264_qsv_encode)
```

Comment:
Missing `@slash.requires(have_ffmpeg_h264_qsv_decode)`.  Same for other tests.

Reply:
added slash.requires for all HW decode dependencies